.gitlab-ci.yml

```diff
@@ -0,0 +1,16 @@
+#variables:
+#    SPEL_VERSION: $(date +%Y.%m.dev%s)
+
+before_script:
+    - echo "STARTING BEFORE_SCRIPT"
+    - apt-get update
+    - apt-get -y install python-pip python-dev build-essential
+    - pip install awscli --upgrade
+    - test $SPEL_CI == "true" && export SPEL_VERSION=$(date +%Y.%m.dev%s)
+
+build:
+    script:
+        - echo "STARTING BUILD"
+        - make
+        - echo "STARTING POST_BUILD"
+        - make post_build
```

Comment:
Can you test that `post_build` executes in gitlab-ci even when the prior `make` fails? That's important so that the cleanup runs no matter what.

Reply:
Removed the need for this test. Separated the post_build phase into its own job.